integration/tests/git_source/HTTPS_proxy_and_git_source.rb

```diff
@@ -64,7 +64,7 @@
 
 #test
 on(master, "#{r10k_fqp} deploy environment -p", :accept_all_exit_codes => true) do |r|
-  regex = /Couldn't resolve proxy 'ilovecatvideos\.com'/
+  regex = /(Couldn't|Could not) resolve proxy.*ilovecatvideos\.com/
```

Comment:
The 'ferritsarebest' regrex has ignore case, but this regrex does not. I think you meant to also have ignore case here?